src/go/rdctl/pkg/snapshot/manager_test.go

```diff
@@ -167,6 +167,14 @@ func TestManager(t *testing.T) {
 		if len(snapshots) != 0 {
 			t.Fatalf("unexpected length of snapshots slice after delete %d", len(snapshots))
 		}
+		err = manager.Delete("no-such-snapshot-id")
```

Comment:
Ideally this would be a separate test case. But if you want to keep it here I'm ok with that, since this test is already unorganized.